README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
```

Comment:
I disagree. This section is about `Initializing Redux`, not about `decorators` which belongs to `Components`. And since there is already the description about `@connect`, I would expect `@provide` to be also there.